collect_app/src/androidTest/java/org/odk/collect/android/feature/AboutPageTest.kt

```diff
@@ -0,0 +1,110 @@
+package org.odk.collect.android.feature
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.R
+import org.odk.collect.android.support.matchers.DrawableMatcher
+import org.odk.collect.android.support.pages.AboutPage
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain.chain
+import org.odk.collect.testshared.RecyclerViewMatcher.Companion.withRecyclerView
+
+@RunWith(AndroidJUnit4::class)
+class AboutPageTest {
```

Comment:
I don't think we need either of these tests really. The "rotate on open source" test is most likely checking some old crash, but we now delegate that view to Google's library and the icons test feels like it's more of a "styling" than a functionality issue.

Reply:
Ok we can remove it.